check/nbformat

```diff
@@ -25,29 +25,33 @@ for arg in $@; do
 done
 
 # Get the working directory to the repo root.
-cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-cd "$(git rev-parse --show-toplevel)"
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )" || exit
+cd "$(git rev-parse --show-toplevel)" || exit
 
 # Install a pinned version of tf-docs
 TF_DOCS_COMMIT=b3dc8a922d8bdc6e998a8ad4f4953359dc6e576a
-TF_DOCS_INSTALLED=$(pip list | grep  -c $TF_DOCS_COMMIT)
+TF_DOCS_INSTALLED="$(pip list | grep -c "$TF_DOCS_COMMIT")"
```

Comment:
shouldn't the internal double quotes be escaped? I'm surprised this works! :)

Reply:
Should it be? AFAIK, this works fine.

Reply:
I confirm that this works, but the outer quotes are unnecessary. The output of the command will be stored in the variable no matter if it has spaces or not. I guess we can keep the internal quotes, but again, unnecessary, especially that we define the TF_DOCS_COMMIT variable above.